.github/workflows/release.yml

```diff
@@ -21,19 +21,19 @@ jobs:
 
       - name: Publish the artifacts to Maven Central
         env:
-          ORG_GRADLE_PROJECT_mavenCentralUsername: ${{ secrets.SONATYPE_NEXUS_USERNAME }}
-          ORG_GRADLE_PROJECT_mavenCentralPassword: ${{ secrets.SONATYPE_NEXUS_PASSWORD }}
+          ORG_GRADLE_PROJECT_mavenCentralUsername: ${{ secrets.MAVEN_CENTRAL_PORTAL_USERNAME }}
+          ORG_GRADLE_PROJECT_mavenCentralPassword: ${{ secrets.MAVEN_CENTRAL_PORTAL_PASSWORD }}
           ORG_GRADLE_PROJECT_signingInMemoryKey: ${{ secrets.SIGNING_PRIVATE_KEY }}
           ORG_GRADLE_PROJECT_signingInMemoryKeyPassword: ${{ secrets.SIGNING_PRIVATE_KEY_PASSWORD }}
-        run: ./gradlew clean build publish --no-daemon --no-parallel
+        run: ./gradlew clean build publishAndReleaseToMavenCentral --no-daemon --no-parallel --no-configuration-cache
```

Comment:
Now we should be able to publish and release from CI. No more manual release is needed.

Reply:
Awesome !! Thank you so much !!